scripts/aptly/aptly_importer.py

```diff
@@ -319,18 +324,19 @@ def run(self):
 
 def main():
     """
-    Usage: python3 aptly_importer.py <config_file>
+    Usage: python3 aptly_importer.py [--dry-run] <config_file>
```

Comment:
Before starting to look code changes I was expecting that `--dry-run` means doing nothing on the system but is only adding `-dry-run` to the import from mirror to repo. Can we rename the argument to make it clearer that dry-run will only be applied to import from mirror to repo?

Reply:
Good call.